zebra-rpc/src/methods/get_block_template_rpcs/types/get_block_template.rs

```diff
@@ -9,6 +9,40 @@ use crate::methods::{
     GetBlockHash,
 };
 
+#[derive(Debug, serde::Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum GetBlockTemplateRequestMode {
+    Template,
+    Proposal,
+}
+
+impl Default for GetBlockTemplateRequestMode {
+    fn default() -> Self {
+        Self::Template
+    }
+}
+
+#[derive(Debug, serde::Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum GetBlockTemplateCapability {
+    LongPoll,
+    CoinbaseTxn,
+    CoinbaseValue,
+    Proposal,
+    ServerList,
+    WorkId,
+}
+
+/// Optional `jsonparametersobject` argument
+#[derive(Debug, serde::Deserialize)]
+pub struct JsonParameters {
+    #[serde(default)]
+    pub mode: GetBlockTemplateRequestMode,
+    #[serde(default)]
+    pub capabilities: Vec<GetBlockTemplateCapability>,
+    pub longpollid: Option<String>,
```

Comment:
If we include the `longpollid` field but ignore it, then clients will expect us to handle it correctly. But we can't do that until we implement #5720.

If we don't list this field, then the RPC will return an error, and the client should stop trying to long poll:
https://en.bitcoin.it/wiki/BIP_0022#Optional:_Long_Polling

Reply:
Updated in [`9209e03`](https://github.com/ZcashFoundation/zebra/pull/5772/commits/9209e031c95a08328530fad2278cea0902abb03c) to return an error saying it's currently unsupported.